src/rp2_common/hardware_uart/include/hardware/uart.h

```diff
@@ -101,6 +101,12 @@ static inline uint uart_get_index(uart_inst_t *uart) {
     return uart == uart1 ? 1 : 0;
 }
 
+static inline uart_inst_t *uart_get_instance(uint instance) {
```

Comment:
Is this change needed by this PR, or was it just pulled in coincidentally?

Reply:
fixed